pkg/kubecost/asset.go

```diff
@@ -755,8 +919,10 @@ func (ca *Cloud) Add(a Asset) Asset {
 	any := NewAsset(start, end, window)
 	any.SetProperties(props)
 	any.SetLabels(labels)
+	any.SetAssetPricingModels(mergePricingModels(ca, a))
 	any.adjustment = ca.Adjustment() + a.Adjustment()
-	any.Cost = (ca.TotalCost() - ca.Adjustment() - ca.Credit) + (a.TotalCost() - a.Adjustment() - ca.Credit)
```

Comment:
This line is the culprit for https://github.com/kubecost/cost-analyzer-helm-chart/issues/1087